manager/state/raft/raft.go

```diff
@@ -986,13 +994,16 @@ func (n *Node) sendToMember(members map[uint64]*membership.Member, m raftpb.Mess
 		defer conn.Conn.Close()
 	}
 
-	_, err := conn.ProcessRaftMessage(ctx, &api.ProcessRaftMessageRequest{Message: &m})
+	_, err := conn.ProcessRaftMessage(ctx, &api.ProcessRaftMessageRequest{Message: &m, Addr: n.Address})
```

Comment:
This will conflict with https://github.com/docker/swarmkit/pull/1121. With that PR, `n.Address` will generally be 0.0.0.0 for a joining node. The manager that they join figures out the address and adds the correct one to the raft log.